tests/integration/plugins/test_signer.py

```diff
@@ -39,6 +39,15 @@ async def verify_signature(records, timestamp, signature):
         await verifier.verify(serialized, signature["signature"], x5u)
 
 
+def fetch_changeset(client, bucket=None, collection=None, **kwargs):
+    bucket = bucket or client.bucket_name
+    collection = collection or client.collection_name
+    endpoint = f"/buckets/{bucket}/collections/{collection}/changeset"
+    kwargs.setdefault("_expected", random.randint(999999000000, 999999999999))
```

Comment:
What does this `_expected` kwarg do? Only curious for my understanding.

Reply:
we use it for cache bust. On client, it contains the highest timestamp of the monitor/changes list, and is received through the push notification.
here we could pass `?_expected=0` it would be fine. Using a random one just makes sure we're not going to be hitting cache during tests